docs/src/configuration/routes/https.md

```diff
@@ -174,7 +168,7 @@ The above cron task will run on the 1st and 15th of the month at 10 am (UTC), an
 It is very important to include the `--no-wait` flag.  If you do not, the cron process will block waiting on the deployment to finish, but the deployment will be blocked by the running cron task.  That will take your site offline until you log in and manually terminate the running cron task.  You want the `--no-wait` flag.  We're not joking.
 {{< /note >}}
 
-The certificate will not renew unless it has less than one month remaining; trying twice a month is sufficient to ensure a certificate is never less than 2 weeks from expiring.  As the redeploy does cause a momentary pause in service we recommend running during non-peak hours for your site.
+The certificate will not renew unless it has less than one month remaining; trying twice a month is sufficient to ensure a certificate is never less than 2 weeks from expiring.
```

Comment:
This is an important caveat.  Please don't remove it.

Though again, this section goes away soon.

Reply:
We already said this a few paragraphs before: L145 : `The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.`

Reply:
ACK.